### PR TITLE
Server Emojis Getting Displayed As <2389238247823> In The Quoter

### PR DESCRIPTION
**Pull request by Renairisu**
Um e-excuse me... Your.... Um Quoter... I Fixed it I think....

### CODE REVIEW

**Comment by DoormatIka** [comment]
what did you just say to me

**Comment by Renairisu** [comment]
I-I said um your quoter has a p-problem when h-handling s-server emojis....

File: src/commands/quote.ts [hunk 1]
```diff
@@ -90,13 +90,18 @@ async function parseQuotes(client: Client, str: string) {
     }
   )
 
+
   const parsed_emotes = await extractObjects(
     parsed_mentions,
-    /(?<=<:(.+?))\d+(?=>)/g,
-    /<:(.+?):\d+>/g,
+    /(?<=<:)(.+?):\d+>/g,
```
Comment (DoormatIka) [inline src/commands/quote.ts]: image
Have you tested the code?

File: src/commands/quote.ts [hunk 2]
```diff
@@ -90,13 +90,18 @@ async function parseQuotes(client: Client, str: string) {
...
+    /<:.+?:\d+>/g,
     async (extracted) => {
-      // placeholder for the replacement algorithm for emojis
-      return ""
+      const emoji = client.emojis.resolve(extracted)
```
Comment (DoormatIka) [inline src/commands/quote.ts]: Why is this here..? You're getting the emojis and returning nothing if an emote is there?

Have you tested this?

**Review by DoormatIka — CHANGES_REQUESTED** [review]
Hmm, try again

File: src/commands/quote.ts [hunk 3]
```diff
@@ -90,20 +90,7 @@ async function parseQuotes(client: Client, str: string) {
...
-    async (extracted) => {
-      const emoji = client.emojis.resolve(extracted)
-      if (!emoji || emoji.guild) {
-        return ""
-      } else {
-        return extracted
-      }
-    }
-  )
+  const parsed_emotes = parsed_mentions.replace(/<a?:[^\s]+:\d+>/g, "")
```
Comment (DoormatIka) [inline src/commands/quote.ts:93]: fair enough

**Comment by DoormatIka** [comment]
I thought you deleted the entire website list I worked tirelessly on 😭

**Review by DoormatIka — APPROVED** [review]
